src/App.jsx

```diff
@@ -19,68 +19,80 @@ function App() {
         <PayPalScriptProvider options={initialOptions}>
         {
           FUNDING_SOURCES.map(fundingSource=>{
-            return(
-              <PayPalButtons
-                fundingSource={fundingSource}
-                key={fundingSource}
-                style={{
-                  layout: 'vertical',// or horizontal
-                  shape: 'rect',
-                  color: (fundingSource === FUNDING.PAYLATER) ? 'gold' : '',
-                }}
-                createOrder={async (data, actions) => {
-                  try {
-                    const response = await fetch("http://localhost:9597/orders", {
-                      method: "POST"
-                    });
-                
-                    const details = await response.json();
-                    return details.id;
-                  } catch (error) {
-                    console.error(error);
-                    // Handle the error or display an appropriate error message to the user
-                  }
-                }}
+              return(
+                <PayPalButtons
+                  fundingSource={fundingSource}
+                  key={fundingSource}
+                  style={{
+                    layout: 'vertical',// or horizontal
+                    shape: 'rect',
+                    color: (fundingSource === FUNDING.PAYLATER) ? 'gold' : '',
+                  }}
+                  createOrder={async (data, actions) => {
+                    try {
+                      const response = await fetch("/api/orders", {
+                        method: "POST",
+                        // use the "body" param to optionally pass additional order information
+                        // like product ids and quantities
+                        body: JSON.stringify({
```

Comment:
I think we also have to pass content type for the backend to know the body payload is json.
```
                  headers: {
                    'Content-Type': 'application/json',
                  },
```